docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
 Directed Acyclic Graphs (Causal Diagrams)
 ------------------------------------------
 
-.. todo::
+.. important::
+
+  The figures represented in this section represent directed acyclic graphs/causal diagrams. These figures are distinct from compartamental model diagrams (such as SIR models). For our purposes we will represent directed acyclic graphs as *circles* connected by arrows and compartamental model diagrams as *squares* connected by arrows.
```

Comment:
could you clarify here if 'for our purposes' applies just to this section, or to the full repo?

Reply:
Yeah, this is slightly confusing because there are no compartmental model diagrams in this section.  Perhaps eventually it would be good to switch the entire repo to this convention, but currently all the cause model diagrams are drawn with circles, which is inconsistent with the description here.

Perhaps we could change the wording here to something like: "DAGs for causal diagrams are commonly represented as *circles* connected by arrows, and compartmental model diagrams are commonly represented as *squares* connected by arrows; we will follow this convention here."